### ISSUE TITLE
Fix up dev, staging and prod config for bot client endpoint

### ISSUE THREAD

**Issue by nicholasamiller** [govlawtech/dva-bot-ui#21]
There's a URL in chatbot-loader.js which is environment-dependent: https://github.com/govlawtech/dva-bot-ui/blob/master/src/scripts/chatbot-loader.js#L38

It should be transformed as follows:
Local dev environment: 'https://neuterbot-client-dev.azurewebsites.net/api/directlineToken'
Staging config: same as above
Prod config: 'https://neuterbot-client.azurewebsites.net/api/directlineToken'

Is there a way to do this with webpack config?

**Comment by sitrobotsit** [comment]
This is done. 

There a now constant API_URL in each .dev .staging and .prod webpack config files which propagates at build time. 

I also noticed that you added  ['INJECTED_BOT_CLIENT_URL'](https://github.com/govlawtech/dva-bot-ui/blob/2b12676df73b4d3edfeb4b299bb9e75cbeb3dfff/src/scripts/va-include.js#L191) to the source var in va-include.js but this key wasn't set anywhere. So I set it to http://localhost:8080/index.html in the dev webpack config, but I wasn't sure what the urls should be in staging and prod, so I just put in 'someurl' and left a TODO to put the real ones in, in both of those files.
